setup.py

```diff
@@ -275,7 +275,7 @@ def get_sphinx_theme_version() -> str:
     'pydruid>=0.4.1',
 ]
 elasticsearch = [
-    'elasticsearch>7, <7.6.0',
+    'elasticsearch>7',
     'elasticsearch-dbapi==0.1.0',
```

Comment:
Should we also bump up `elasticsearch-dbapi`? Current version is `0.2.4`

Reply:
> Should we also change the lower bound to `>=7.6.0`?
I think we do not want to limit the lower bound in this case. I am not even sure what limit it should be to be honest :). Various problems with Python 3.9 have been solved in I think few versions between 7.5.1 and 7.13.1 (including one solved in 7.10.1) . I think we do not want to "force" people to upgrade if they don't want in this case. We could potentially add some limit for "python>=3.9" but I think this has marginal impact. Most people installing it from the scratch will have latest version and if they use constraints, even when upgrading they will bump it.

> Should we also bump up elasticsearch-dbapi? Current version is 0.2.4

No idea. There is no explanation on why those limitations were added so I have no idea what effect it will have to remove it. I am summoning @jedcunningham here :)